recipes/stages/segmentation/class_incr.py

```diff
@@ -5,10 +5,11 @@
 ]
 
 optimizer = dict(
-    type='SGD',
+    _delete_=True,
+    type='Adam',
```

Comment:
Adam shows better results than SGD? I remember you said SGD is better than Adam.

Reply:
There was slight accuracy gain w/ 120 image set, but it affected too slow training time w/ 12 image set. With the small set, Adam is giving very fast learning.

Reply:
Okay, thank you for detailed explanation. Could you confirm grad_norm for segmentation tasks?
I remember OTEseg team applied Adam optimizer to stabilize grad_norm of very early training.

Reply:
grad_norm works w/ Adam, and does not work w/ SGD.